src/kolmogorov_solve.jl

```diff
@@ -0,0 +1,53 @@
+struct NNKolmogorov{C,O,S} <: NeuralNetDiffEqAlgorithm
+    chain::C
+    opt::O
+    sdealg::S
+   
+end
+NNKolmogorov(chain  ; opt=Flux.ADAM(0.1) , sdealg = EM()) = NNKolmogorov(chain , opt , sdealg)
+ 
+function DiffEqBase.solve(
+    prob::KolmogorovPDEProblem,
+    alg::NeuralNetDiffEqAlgorithm;
+    abstol = 1f-6,
+    verbose = false,
+    maxiters = 300,
+    save_everystep = false,
+    dt,
+    kwargs...
+    )
+
+    tspan = prob.tspan
+    phi = prob.phi
+    xspan = prob.xspan
+    sigma = prob.sigma
+    μ = prob.μ
+    d = prob.d
+    ts = tspan[1]:dt:tspan[2]
+    N = size(ts)
+    T = tspan[2]
+    #hidden layer
+    chain  = alg.chain
+    opt    = alg.opt
+    sdealg = alg.sdealg
+    ps     = Flux.params(chain)
+    xi     = rand(Uniform(xspan[1] , xspan[2]), d , N[1] )
```

Comment:
Do this directly instead of using `Uniform` and drop the dependency: it'll just make things more complex.

Reply:
How do I do this directly? Take an input in the problem?